EmbedScratch.php

```diff
@@ -58,8 +59,15 @@ function renderEmbedScratch ($input, $argv, $parser) {
 		{
 			$height = $argv['height'];
 		}
+		if (
+			!empty( $argv['menu'] ) &&
+			settype( $argv['menu'], 'integer' )
```

Comment:
What's the purpose of this?

Reply:
Hiding the blue menu (margin-top:-52px) or not.

Reply:
This blue menu?
![image](https://user-images.githubusercontent.com/28599280/56024911-a097b680-5d43-11e9-808d-ac51fa94c82a.png)
I don't think that should ever appear... why is an option necessary?

Reply:
No, not this menu :)
The one with the green flag. If you've a project that doesn't rely on the flag, you should be able to hide it. See both screenshots below.
![menu](https://user-images.githubusercontent.com/10480556/56027349-2dc41680-5d16-11e9-913c-c9b798c9b3eb.jpg)
![nomenu](https://user-images.githubusercontent.com/10480556/56027351-2dc41680-5d16-11e9-8271-41d19ba892cc.jpg)

Reply:
It's more greyish, not blue. Sorry for that confusion...

Reply:
Ah, I see. I suppose that's good to have.